web_tests/pages/change_password_page.py

```diff
@@ -0,0 +1,135 @@
+
+    @property
+    def change_password_button(self):
+        return self.wait_for_element(ChangePassword.CHANGE_PASSWORD_BUTTON)
+
+    @property
+    def update_password_form(self):
+        return self.wait_for_element(ChangePassword.UPDATE_PASSWORD_FORM)
+
+    @property
+    def current_password_field(self):
+        return self.wait_for_element(ChangePassword.CURRENT_PASSWORD_FIELD)
+
+    @property
+    def new_password_field(self):
+        return self.wait_for_element(ChangePassword.NEW_PASSWORD_FIELD)
+
+    @property
+    def confirm_password_field(self):
+        return self.wait_for_element(ChangePassword.CONFIRM_NEW_PASSWORD_FIELD)
+
+    @property
+    def cancel_button(self):
+        return self.wait_for_element(ChangePassword.CANCEL_BUTTON)
+
+    @property
+    def save_button(self):
+        return self.wait_for_element(ChangePassword.SAVE_BUTTON)
+
+    @property
+    def error_message_for_current_password_field(self):
+        return self.wait_for_element(ChangePassword.ERROR_MESSAGE_FOR_CURRENT_PASSWORD)
+
+    @property
+    def error_message_for_new_password_field(self):
+        return self.wait_for_element(ChangePassword.ERROR_MESSAGE_FOR_NEW_PASSWORD)
+
+    @property
+    def error_message_for_confirm_password_field(self):
+        return self.wait_for_element(ChangePassword.ERROR_MESSAGE_FOR_CONFIRM_PASSWORD)
+
+    @property
+    def error_passwords_do_not_match(self):
+        return self.wait_for_element(ChangePassword.ERROR_PASSWORDS_DO_NOT_MATCH)
+
+    @property
+    def error_minimum_number_of_characters(self):
+        return self.wait_for_element(ChangePassword.ERROR_MINIMUM_NUMBER_OF_CHARACTERS)
+
+    @property
+    def error_no_letter_in_password(self):
+        return self.wait_for_element(ChangePassword.ERROR_NO_LETTER_IN_PASSWORD)
+
+    @property
+    def current_password_is_incorrect_toast(self):
+        return self.wait_for_element(ChangePassword.CURRENT_PASSWORD_INCORRECT)
+
+
+    def change_password_page(self):
+        self.dropdown_menu_button.click()
+        self.change_password_button.click()
+        return self
+
+    def change_password_form_is_displayed(self):
+        return self.update_password_form.is_displayed()
+
+    def save_button_click(self):
+        self.save_button.click()
+
+    def empty_form_cant_be_send(self):
+        self.save_button_click()
+        return self.change_password_form_is_displayed()
+
+    def error_message_for_current_password_is_displayed(self):
+        return self.error_message_for_current_password_field.is_displayed()
+
+    def error_message_for_new_password_is_displayed(self):
+        return self.error_message_for_new_password_field.is_displayed()
+
+    def error_message_for_confirm_password_is_displayed(self):
+        return self.error_message_for_confirm_password_field.is_displayed()
+
+    def current_password_is_incorrect_toast_is_displayed(self, password):
+        self.current_password_field.send_keys(password.current_password)
```

Comment:
Note that all 4 methods below are the same, they differ only by entered data. So its' code duplication.
Checks should be performed in separate methods. Above you add some methods to check if some element is displayed but didn't use them